openstack/designate/values.yaml

```diff
@@ -92,17 +92,6 @@ memcached:
   metrics:
     enabled: true
 
-rabbitmq:
```

Comment:
the rabbitmq is not ours. Please do not touch

Reply:
reverting